openaerostruct/docs/advanced_features/scripts/basic_2_sec.py

```diff
@@ -0,0 +1,237 @@
+# docs checkpoint 0
```

Comment:
It shouldn't but none of the examples in the examples folder have doc checkpoint references like this one does. I can put copies in there though.

Reply:
Ok, I don't mind having doc checkpoints in the example file or keeping these in the current folder under docs as long as users can easily find them. And since you embedded all three files in the rst files it's good.
Either way, we should avoid putting two copies in difference places because that will make future maintenance difficult.